docusaurus/docs-internals/editor-controls/unit-controls/slider.md

```diff
@@ -5,39 +5,57 @@ toc_max_heading_level: 4
+| `config?.debounceUpdate` | `boolean`                                                                                                                                                                                  |   `false`    | Determines whether the UI is updated using debouncing instead of throttling                                                                                                                                                                                                                                                                                                                                                       |
+| `config?.updateRate`     | `number`                                                                                                                                                                                   |     `50`     | The delay in milliseconds after which the UI responds to value changes                                                                                                                                                                                                                                                                                                                                                            |
+| `default?`               | `Default`                                                                                                                                                                                  |      -       | The default control value. <br/> <br/> <b>`Default: { value: number; suffix: string; }`</b> <br/> <br/> `value` - the default number of the control  <br/> `suffix` - your custom or CSS unit (`"px"`, `"%"`, `"em"` and others), will be active in the dropdown if multiple units are available                                                                                                                                  |
+| `style?`                 | `function`                                                                                                                                                                                 |      -       | This function generates CSS output based on the value from the control. The parameter is an object containing a `value` key, which holds the current value of the control. The function returns an object with a CSS selector key and CSS property values. <pre>`style: ({value}) => {`<br/> `return {`<br/> ` "{{WRAPPER}} .selector": {`<br/> `   width: ${value.value}${value.unit} \|\| "%"`<br/> ` }`<br/> `}`<br/>`}`</pre> |
```

Comment:
1. about: `width: ${value.value}${value.unit} || "%"`
let's use: `width: value.value.toString() + (value.unit ?? "%" )`
2. and instead of  `.selector` set as example: `.brz-map`